core/templates/pages/certificate-assessment-player-page/certificate-assessment-conversation-skin.component.css

```
@@ -0,0 +1,169 @@
+  .certificate-assessment-primary-button {
+    background-color: #00645c;
+    border: none;
+    border-radius: 6px;
+    color: #fff;
+    font-family: 'Roboto', sans-serif;
+    font-size: 14px;
+    font-weight: 500;
+    margin-top: 20px;
+  }
+
+  .certificate-assessment-conversation-skin ::ng-deep .oppia-progress-bar {
```

Stylelint (17.14.1)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)